src/get-started/create-service.md

```diff
@@ -739,14 +770,13 @@ To create the second wallet put the code into `create-wallet-2.json` file:
 ```json
 {
   "body": {
-    "pub_key": "d1e877472a4585d515b13f52ae7bfded1ccea511816d7772cb17e1ab20830819",
-    "name": "Janie Roe"
+    "pub_key":"ae6a1c4e84886999dfec7f4d792bf133e7beacf974c000fe45c443727df49df2",
```

Comment:
Please add spaces after a semicolon here and in other places where they are absent.